src/components/ChannelList/ChannelList.tsx

```diff
@@ -380,18 +398,29 @@ const UnMemoizedChannelList = <
     },
   );
 
-  const showChannelList = !searchActive || additionalChannelSearchProps?.popupResults;
+  const showChannelList =
+    (!searchActive && !searchIsActive) || additionalChannelSearchProps?.popupResults;
```

Comment:
Not sure what the difference between `searchActive` and `searchIsActive` is at first glance.

Reply:
One is a label to display the old ChannelSearch and the other is derived from the state of searchController to display the new Search component.